demos/blast/blast_in_example.cpp

```diff
@@ -0,0 +1,43 @@
+#include <iostream>
+#include <seqan/blast.h>
+
+using namespace seqan;
+
+int main()
+{
+    typedef Align<String<AminoAcid>, ArrayGaps> TAlign;
+    typedef BlastMatch<CharString, CharString, uint32_t, TAlign> TBlastMatch;
+    typedef BlastRecord<CharString, CharString, uint32_t, TAlign> TBlastRecord;
+    typedef BlastIOContext<Blosum62> TContext;
+
+    std::string inPath = std::string(SEQAN_PATH_TO_ROOT()) + "/tests/blast/plus_header_defaults.blast";
+
+    BlastTabularIn<TContext> in(toCString(inPath));
```

Comment:
> Should be called just BlastFileIn.

There are two formats currently implemented: BlastTabular and BlastReport. I could rename the FormattedFiles from BlastTabularOut to BlastTabularFileOut a.s.o
Do you mean that?

> You should catch the exceptions thrown by the constructor.

Yes!